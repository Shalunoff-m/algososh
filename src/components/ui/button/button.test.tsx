import renderer from 'react-test-renderer';
import { Button } from './button';
import { render, screen, fireEvent } from '@testing-library/react';
import { Direction } from '../../../types/direction';

describe('Проверка кнопки', () => {
  it('Кнопка с текстом', () => {
    const tree = renderer.create(<Button text='Текст кнопки' />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
