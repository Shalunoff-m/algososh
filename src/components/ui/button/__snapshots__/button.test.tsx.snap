// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Проверка кнопки Кнопка с текстом 1`] = `
<button
  className="text text_type_button text_color_primary button undefined false "
  type="button"
>
  <p
    className="text undefined"
  >
    Текст кнопки
  </p>
</button>
`;
